src/NuGet.Services.Metadata.Catalog.Monitoring/Notification/AggregateMonitoringNotificationService.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class AggregateNotificationService : IMonitoringNotificationService
+    {
+        IEnumerable<IMonitoringNotificationService> _notificationServices;
+
+        public AggregateNotificationService(IEnumerable<IMonitoringNotificationService> notificationServices)
+        {
+            _notificationServices = notificationServices;
+        }
+
+        public Task OnPackageValidationFinished(PackageValidationResult result, CancellationToken token)
```

Comment:
OnPackageValidationFinishedAsync, if possible. Same for OnPackageValidationFailed.